healthcheck.py

```diff
@@ -0,0 +1,25 @@
+import sys
```

Comment:
I'm not sure why this file was added

Reply:
This is also part of that test script -- this is used as a healthcheck in the docker-compose script so the tests run only after the test agents have reached a healthy state. I think this file could be legitimately useful to add to the project (don't have to rely on coming up with a curl command to do the same kind of check from a container orchestrator), though it probably belongs in the `docker` directory.